src/GitVersion.Core/GitVersion.Core.csproj

```diff
@@ -8,10 +8,15 @@
         <PackageId>GitVersion.Core</PackageId>
         <Title>GitVersion</Title>
         <Description>Derives SemVer information from a repository following GitFlow or GitHubFlow. This is the Core library which both GitVersion cli and Task use allowing programatic usage of GitVersion.</Description>
-
         <Product>$(AssemblyName)</Product>
     </PropertyGroup>
 
+    <ItemGroup>
```

Comment:
I would rather expose the class as public instead of exposing all the internals. Just change the `OperationWithExponentialBackoff` to `public`

Reply:
Done